Gemfile.lock

```diff
@@ -81,7 +81,7 @@ GEM
     daemons (1.2.4)
     diff-lcs (1.5.1)
     docile (1.1.5)
-    dotenv (2.8.1)
+    dotenv (3.1.2)
```

Comment:
initially I thought `dotenv` was the issue. Later it turned out not to be but I already had it updated. I see no harm in keeping it that way.